packages/constants/assets/index.ts

```diff
@@ -2,7 +2,14 @@ import { Metadata } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/
 import LocalAssetRegistry from './local-asset-registry.json';
 import { JsonValue } from '@bufbuild/protobuf';
 
+export const localAssets: Metadata[] = LocalAssetRegistry.map(a =>
+  Metadata.fromJson(a as JsonValue),
+);
+
 export const STAKING_TOKEN = 'penumbra';
+export const STAKING_TOKEN_METADATA = localAssets.find(
+  metadata => metadata.display === STAKING_TOKEN,
+);
```

Comment:
Maybe add a `!` as we guarantee it's there? It would be nice if Typescript had the Rust-like: `.ok_or_else(Error::Something("no there"))`

Reply:
Whoops, good call. I'll do that and throw an error if it's falsey.

Reply:
JK re: throwing an error — TypeScript complained if I checked for its presence after already asserting its presence. But I've at least added the `!`.